docs/source/managed-federation/overview.mdx

```diff
@@ -3,7 +3,7 @@ title: Managed federation overview
 sidebar_title: Overview
 ---
 
-[Apollo Studio](https://www.apollographql.com/docs/studio/) provides free **managed federation** features for data graphs that use Apollo Federation.
+[Apollo Studio](https://www.apollographql.com/docs/studio/federated-graphs/) provides free **managed federation** features for data graphs that use Apollo Federation.
```

Comment:
How about this?:

```suggestion
Apollo Studio provides free [**managed federation** features](https://www.apollographql.com/docs/studio/federated-graphs/) for data graphs that use Apollo Federation.
```

Honestly, I'm less compelled to click on "Apollo Studio" than I am to click on the managed federation features link anyhow.  I don't think we need multiple links given that the landing page suggested here has a video and text on it explaining the offering.